tools/test.py

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env python3
+
```

Comment:
Let's call this this file `pytest_wrapper` (or something else) to avoid name collision with the CPython test module.